jibri/rootfs/etc/cont-init.d/11-get-https-cert

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/with-contenv bash
+
+while ! openssl s_client -showcerts -connect meet.jitsi:443 < /dev/null | openssl x509 -outform PEM > /usr/local/share/ca-certificates/meet.crt; do sleep 1; done
```

Comment:
This won't work if the port is not 443. I know there are other problems in that case, but we should use the env var here.